kornia/contrib/extract_patches.py

```diff
@@ -125,20 +126,25 @@ class CombineTensorPatches(nn.Module):
 
     def __init__(
         self,
+        original_size: Union[int, Tuple[int, int]],
         window_size: Union[int, Tuple[int, int]],
         unpadding: Union[int, Tuple[int, int]] = 0,
```

Comment:
missing `stride`

Reply:
`combine_tensor_patches` currently only supports `stride == window_size`. Refer [here](https://github.com/kornia/kornia/blob/62f114e7883292ae6bb38874cfaca2d98c13a160/kornia/contrib/extract_patches.py#L165).

Reply:
Might be worth to add a note somewhere about this detail

Reply:
We could add the stride param and forward it to `combine_tensor_patches` where a `NotImplementedError` will be thrown.